example/bfp_demo/CMakeLists.txt

```diff
@@ -10,15 +10,30 @@ add_executable( ${APP_NAME} )
 
 target_sources( ${APP_NAME}
   PRIVATE
-    main.c 
+    main.c
 )
 
 target_compile_options( ${APP_NAME}
-  PRIVATE 
-    -Os -g -Wno-format -Wno-unused-variable
+  PRIVATE
+    -Os
     $<$<PLATFORM_ID:XCORE_XS3A>:-fxscope>
 )
 
+# Add options for different compilers
+if ( NOT CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+  target_compile_options( ${APP_NAME}
+    PRIVATE
+      -g
+      -Wno-unused-variable
+      -Wno-format
```

Comment:
Do we need these 2 flags in all example applications?

Reply:
I removed them.